chain/x/zoracle/internal/types/request.go

```diff
@@ -1,32 +1,39 @@
 package types
 
 import (
-	"fmt"
-	"strings"
+	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
-// Request is a type to store detail of request
+// Request is a type to store detail of request.
```

Comment:
Request is a data structure that stores the detail of a request to an oracle script.